templates/cluster/aws-standalone-cp/templates/k0scontrolplane.yaml

```diff
@@ -15,10 +15,21 @@ spec:
       - --enable-cloud-provider
       - --kubelet-extra-args="--cloud-provider=external"
       - --disable-components=konnectivity-server
+    {{- $certs := dict "registry.crt" $global.registryCertSecret "k0s-url.crt" $global.k0sURLCertSecret }}
     {{- $auth := .Values.k0s.auth }}
     {{- $files := .Values.k0s.files }}
-    {{- if or (and $auth $auth.enabled) $files }}
+    {{- if or (and $auth $auth.enabled) $files $global.registryCertSecret $global.k0sURLCertSecret }}
     files:
+      {{- range $path, $secret := $certs }}
+      {{- if $secret }}
+      - contentFrom:
+          secretRef:
+            name: {{ $secret }}
+            key: tls.crt
+        permissions: "0664"
+        path: /usr/local/share/ca-certificates/{{ $path }}
+      {{- end }}
+      {{- end }}
```

Comment:
This one is tricky to decide its place: if we put it in the first place, potentially this entry could get rewritten (i am not sure though about the order of processing the `files` list). might it be a good idea to put the cert either after the `auth.config` or even push it to the very end?